### PR TITLE
Dynamic security analysis server configuration

### PR DESCRIPTION
**Pull request by thangqp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +208,7 @@ http://localhost:5028/swagger-ui.html  // report-server
 http://localhost:5029/swagger-ui.html  // explore-server
 http://localhost:5036/swagger-ui.html  // dynamic-mapping-server
 http://localhost:5032/swagger-ui.html  // dynamic-simulation-server
+http://localhost:5034/swagger-ui.html  // dynamic-security-analysis-server
```
Comment (etiennehomer) [inline README.md:211]:
```suggestion
http://localhost:5040/swagger-ui.html  // dynamic-security-analysis-server
```

**Review by etiennehomer — CHANGES_REQUESTED** [review]
Deploy on Azure or not ?
see k8s/live/azure-integ/kustomization.yaml

File: README.md (hunk 1 above)
Comment (etiennehomer) [reply README.md:211]: user-identity-server took 5034

**Comment by thangqp** [comment]
> Deploy on Azure or not ? see k8s/live/azure-integ/kustomization.yaml
YES, but waiting image released

File: README.md (hunk 1 above)
Comment (thangqp) [reply README.md]: DONE in https://github.com/gridsuite/deployment/pull/448/commits/081f44077c2802e01a9419c1e8151784ef0b749f

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**